src/plus/integrations/models/gitHostIntegration.ts

```diff
@@ -663,10 +683,13 @@ export abstract class GitHostIntegration<
 				cancellation,
 				silent,
 			);
+			this.resetRequestExceptionCount('searchMyPullRequests');
 			return { value: pullRequests, duration: Date.now() - start };
 		} catch (ex) {
-			Logger.error(ex, scope);
-			return { error: ex, duration: Date.now() - start };
+			return this.handleProviderException('searchMyPullRequests', ex, scope, {
```

Comment:
Sorry I'm catching this a bit late, but: we moved `searchMyPullRequests` off of `handleProviderException` specifically because it results in an error notification in VS Code if the user errors too many times, and folks whose integration was having issues with Launchpad (due to too many PRs, large/complex repos, etc.) were seeing the notification all the time since Launchpad query automatically runs in an interval.

We need to make sure we do not send a message in this case.